epaint/src/color.rs

```diff
@@ -186,6 +186,36 @@ impl Color32 {
     }
 }
 
+#[macro_export]
```

Comment:
```suggestion
/// Construct a [`Color32`] from a hex RGB or RGBA string.
///
/// ```
/// assert_eq!(hex_color!("#202122"), Color32::from_rgb(0x20, 0x21, 0x22));
/// assert_eq!(hex_color!("#abcdef12"), Color32::from_rgba_unmultiplied(0xab, 0xcd, 0xef, 0x12));
/// ```
#[macro_export]
```

Reply:
Done!